.github/workflows/build.yml

```diff
@@ -419,6 +419,38 @@ jobs:
       - name: Compute test coverage
         uses: codecov/codecov-action@v3
 
+  job_deno_unit_tests:
+    name: Deno Unit Tests
+    needs: [job_get_metadata, job_build]
+    timeout-minutes: 10
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Check out current commit (${{ needs.job_get_metadata.outputs.commit_label }})
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ env.HEAD_COMMIT }}
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: 'package.json'
+      - name: Set up Deno
+        uses: denoland/setup-deno@v1
```

Comment:
l: I know we don't do this everywhere else but we probably should: Would you mind pinning this to an exact version?